COPYING

```diff
@@ -0,0 +1,340 @@
+
```

Comment:
This is confusing. You should have this in a BSD licensed project

Reply:
It's a GNU license on the test resources. This is explained at the top of the LICENSE file (I copied the way they did it for python-bioformats)